File: fixtures/scale/csharp/large/Families/Keyword/KeywordVectors.cs
using System.Collections.Generic;
using CipherLab.Abstractions;

namespace CipherLab.Families.Keyword
{
    public static class KeywordVectors
    {
        public static List<TestVector> All()
        {
            return new List<TestVector>
            {
                new TestVector("ATTACKATDAWN", "FZAILULFQOLD"),
                new TestVector("THEQUICKBROWNFOX", "YNLYDSNWOFDMEXHR"),
                new TestVector("DEFENDTHEEASTWALL", "IKMMWNETRSPIKOTFG"),
            };
        }
    }
}

File: fixtures/scale/php/large/src/Families/Keyword/KeywordVectors.php
<?php

namespace CipherLab\Families\Keyword;

use CipherLab\Abstractions\TestVector;

// Known-answer test data captured from KeywordCipher's own encode().
class KeywordVectors
{
    /** @return TestVector[] */
    public static function all(): array
    {
        return [
            new TestVector("ATTACKATDAWN", "FZAILULFQOLD"),
            new TestVector("THEQUICKBROWNFOX", "YNLYDSNWOFDMEXHR"),
            new TestVector("DEFENDTHEEASTWALL", "IKMMWNETRSPIKOTFG"),
        ];
    }
}
